nixos/tests/corteza.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, ... }:
+let
+  port = 8080;
+in
+{
+  name = "corteza";
+  meta.maintainers = with lib.maintainers; [ prince213 ];
```

Comment:
Maybe add the NGI maintainers here and in the module, so all of us get pinged in case something breaks.